Updated description for VEBA in Docs (#601)
vmware-samples/vcenter-event-broker-appliance: README.md

## Overview

> The [VMware Event Broker Appliance](https://flings.vmware.com/vmware-event-broker-appliance#summary) Fling enables customers to unlock the hidden potential of events in their SDDC to easily create [event-driven automation](https://octo.vmware.com/vsphere-power-event-driven-automation/). The VMware Event Broker Appliance includes support for vCenter Server and VMware Horizon event as well as non-VMware events through native webhook event provider.Extending these existing events by easily triggering custom or prebuilt actions to deliver powerful integrations within your datacenter across public cloud has never been more easier before. A detailed list of use cases and possibilities with VMware Event Broker Appliance is available [here](https://vmweventbroker.io)

Reviewer: ```suggestion
The [VMware Event Broker Appliance](https://flings.vmware.com/vmware-event-broker-appliance#summary) Fling enables customers to unlock the hidden potential of events in their SDDC to easily create [event-driven automation](https://octo.vmware.com/vsphere-power-event-driven-automation/). The VMware Event Broker Appliance includes support for vCenter Server and VMware Horizon events as well as any valid `CloudEvent` through the native webhook event provider. Easily triggering custom or prebuilt actions to deliver powerful integrations within your datacenter across public cloud has never been more easier before. A detailed list of use cases and possibilities with VMware Event Broker Appliance is available [here](https://vmweventbroker.io)
```
